test/extension/numeric/Jamfile

```diff
@@ -25,3 +25,4 @@ run kernel.cpp ;
 compile-fail kernel_1d_fixed_even_size_fail.cpp ;
 run matrix3x2.cpp ;
 run numeric.cpp ;
+convolve_2d.cpp ;
```

Comment:
The `run` is missing and causing this failure at https://ci.appveyor.com/project/stefanseefeld/gil/builds/26526874/job/8lqgesnkn6bhs26c

```
libs\gil\test\extension\numeric\Jamfile:28: in modules.load
ERROR: rule "convolve_2d.cpp" unknown in module "Jamfile<C:\projects\boost-root\libs\gil\test\extension\numeric>"
```